lib/internal/modules/esm/loader.js

```diff
@@ -107,6 +108,24 @@ class ESMLoader {
    */
   translators = translators;
 
+  constructor(isInternal = false) {
+console.log({ isInternal })
```

Comment:
This console.log looks out of place.

Reply:
It's debug code for Geoffrey: the test is failing and it does not behave as the real-world does (but we don't yet know why—it seems it should). See this thread: https://github.com/nodejs/node/pull/42314#discussion_r825324802